src/app/components/experiment/experiment.component.sass

```diff
@@ -4,13 +4,14 @@
     justify-content: space-between
     align-items: center
 
-.clone-button
-    padding-block: 0.3em
-    height: auto
+.buttons
+    display: flex
+    flex-direction: row-reverse
```

Comment:
Do not reverse positioning with css if possible. Order in dom should correspond to visual order of items.